collections/_article/godot-4-0-sets-sail.md

```diff
@@ -0,0 +1,685 @@
+
+![An island seen from above with a calm ocean extending to the horizon](/storage/blog/godot-4-0-sets-sail/02-3d-outdoor-with-editor.webp)
+
+While we're super excited to leverage Vulkan for performance optimization going forward, it was very important not to penalize users with less powerful hardware. For that, we've also integrated an OpenGL-based compatibility renderer aimed at supporting older and lower-end devices. As always, we want to see people create games on the full spectrum of devices for everyone to enjoy.
+
+We're also working on a [Direct3D 12](https://github.com/godotengine/godot/pull/64304) renderer for better Windows and Xbox support.
+
+With [Je06jm](https://github.com/Je06jm)‘s contribution, you can now also take advantage of AMD's [Fidelity FX Super Resolution 1.0](https://github.com/godotengine/godot/pull/51679) (FSR 1.0) to dynamically and beautifully render at lower resolution while keeping your game running smoothly. Spoiler: Support for FSR 2.1 is planned in future releases.
+
+<a id="highly-improved-lighting-shadows"></a>
+### Highly Improved Lighting & Shadows
+
+For starters, Godot's global illumination systems have been remade from scratch.
+
+For the first time ever, Godot 4 introduces a novel real-time global illumination technique for large open worlds. **SDFGI** - or Signed Distance Field Global Illumination was created and implemented by Juan and you can learn a lot more about it [here](https://godotengine.org/article/godot-40-gets-sdf-based-real-time-global-illumination).
+
+![GDBot the robot standing face camera in front of two trees](/storage/blog/godot-4-0-sets-sail/02-3d-gi-sdfgi.webp)
+
+GIProbe has been replaced by the **VoxelGI** node, a real-time solution fit for small and medium-scale environments with particularly good results for interiors. Of course, you can still use lightmaps to pre-render lighting and shadows on low-end devices but lightmap baking now uses the GPU for much faster rendering.
+
+![Interior of a tiny but well-lit apartment with only a living room and a kitchen](/storage/blog/godot-4-0-sets-sail/02-3d-gi-voxelgi.webp)
+
+Lastly, we were never quite satisfied with how shadows looked in Godot 3. Godot 4 was an opportunity to go back to the drawing board on shadow rendering to achieve higher quality and provide more granular control.
+
+<video autoplay loop muted playsinline>
+  <source src="/storage/blog/godot-4-0-sets-sail/02-3d-shadows.mp4" type="video/mp4">
+</video>
+
+<a id="new-rendering-optimization-techniques"></a>
+### New Rendering Optimization Techniques
+
+Godot 4 puts at your disposal several new rendering optimization techniques, made possible by Joan Fons ([JFonS](https://github.com/JFonS)) and Juan.  
+In addition to manual [occlusion culling](https://github.com/godotengine/godot/pull/48050) which we backported to Godot 3.5, the new automatic occlusion culling can scan your scene and detect models hidden by other geometry.
```

Comment:
The occlusion culling used in 4.0 was not backported to 3.x. Likewise, the occlusion culling in 3.x was not forward-ported to 4.0. 

```suggestion
The new automatic [occlusion culling](https://github.com/godotengine/godot/pull/48050) can detect models hidden by other geometry and dynamically remove them to increase both CPU and GPU rendering performance.
```

Reply:
If this part is edited, please either remove the line break or add another line so it looks like a proper paragraph.